packages/rocketchat-lib/server/startup/settingsOnLoadCdnPrefix.js

```diff
@@ -0,0 +1,13 @@
+/* globals WebAppInternals*/
+RocketChat.settings.onload('CDN_PREFIX', function(key, value) {
+	if (_.isString(value || false)) {
+		return typeof WebAppInternals !== 'undefined' && WebAppInternals !== null && WebAppInternals.setBundledJsCssPrefix(value);
+	}
+});
+
+Meteor.startup(function() {
+	const value = RocketChat.settings.get('CDN_PREFIX');
+	if (_.isString(value || false)) {
```

Comment:
Ok, but that looks strange and will allow spaces. What about `if (_.isString(value) && value.trim())` just to be more clear?